practice-two/src/styles/sections/plan.css

```diff
@@ -0,0 +1,173 @@
+.section-plan {
+  background-color: var(--background-color-primary);
+}
+
+.plan-container {
+  padding-top: 52px;
+  padding-bottom: 42px;
+  text-align: center;
+}
+
+.plan-heading {
+  margin-bottom: 20px;
+  font-size: var(--font-size-lg);
+  color: var(--color-text-primary);
+}
+
+.choose-plan {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  align-items: center;
+  gap: 24px 0;
+}
+
+.plan-card-silver {
+  background: linear-gradient(
+    224deg,
+    var(--background-color-translucent-white) 5.13%,
+    var(--background-color-transparent-white) 90.88%);
+  color: var(--color-text-primary);
+}
+
+.plan-card-gold {
+  background-color: var(--background-color-card-gold);
+  color: var(--color-text-secondary);
+}
+
+.plan-card-platinum {
+  background: linear-gradient(
+    223deg,
+    var(--background-color-transparent-black) 4.85%,
+    var(--background-color-translucent-black) 96.5%);
+  color: var(--color-text-primary);
+}
+
+.card-heading,
+.plan-duration {
+  font-size: var(--font-size-xsmall);
+  line-height: var(--line-height-2lg);
+  letter-spacing: var(--letter-spacing-md);
+}
+
+.card-heading {
+  font-family: 'lucida grande', sans-serif;
```

Comment:
Should define font face

Reply:
I've already defined that font-face in my project, you can check it in styles/base/typography.css